NEWS: update for 2.0.2 release

hppritcha opened the pull request:
Fixes #2239

@jsquyres 

Signed-off-by: Howard Pritchard <howardp@lanl.gov>

jsquyres commented on NEWS:
```diff
@@ -53,6 +53,37 @@ included in the vX.Y.Z section and be denoted as:
+  Limin Gu for reporting.
+- Fix a problem when using MPI dynamic memory windows.  Thanks to
+  Christoph Niethammer for reporting.
+- Fix a problem with Open MPI's pkgconfig files.  Thanks to amckinstry
```
@amckinstry is Alastair McKinstry; let's use his name.

hppritcha commented:
@jsquyres put in Alastair's full name, added some new bullets for more recent commits reported by users, added some verbosity to a some of the bullets to help with google search.

hppritcha commented:
I changed the Valgrind spelling somewhat.  Capitalized if used as a proper noun, otherwise, lower case if used as an adjective or adverb.

hppritcha commented:
updated with info relevant to PR #2529

hppritcha commented:
@jsquyres updated to include PR #2556

hppritcha commented:
Have to careful with commits from the web interface to manually add in a sign off line.

jsquyres commented:
@hppritcha Did you want to squash these down?
